docker/install/ubuntu_install_sbt.sh

```diff
@@ -25,7 +25,7 @@ set -o pipefail
 apt-get update && apt-get install -y apt-transport-https
 
 # Install the necessary dependencies for sbt
-echo "deb https://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list
+echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list
```

Comment:
```suggestion
echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | sudo tee /etc/apt/sources.list.d/sbt.list
echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | sudo tee /etc/apt/sources.list.d/sbt_old.list
```

Reply:
Sorry I haven't had time to update the docs but the new Debian packages are now under `all main`.

Reply:
Not a problem, thanks for pointing it out :smile_cat: